### DISCUSSION TITLE
Pause streaming chart

### DISCUSSION THREAD

**Discussion by herzplatinedurchgebrannt** [Megabit/Blazorise#4257] [Q&A]
Hey guys,

is there a way to pause streaming charts?

chartjs offers an pause bool in the options.
https://nagix.github.io/chartjs-plugin-streaming/latest/guide/options.html#onrefresh

Unfortunately I cannot find this option in Blazorise.

Thank you in advance!

**Comment by stsrki** [comment]
It is not possible at the moment. I'm working on a new plugin for charts, #4247, and I also noticed that the streaming chart has some new options like `pause`. We can make it for the next v1.2

**Reply by herzplatinedurchgebrannt** [reply]
Thanks for your answer!
